server/controllers/customerController.js

```diff
@@ -0,0 +1,39 @@
+const { Retailer, Customer, Reservation } = require("../models");
+
+const customerController = {
+  getAllRetailers(req, res) {
+    console.log("get all retail route works");
+    Retailer.find(
+      {},
+      "retailerId retailerName address phoneNumber storeHours capacity timers"
+    ).then((results) => {
+      if (!results)
+        res.status(404).json({ message: "No retailers found in database" });
+
+      res.json(results);
+    });
+  },
+  searchRetailers({ params, body }, res) {
+    Retailer.find({ retailerName: new RegExp(params.searchTerm, "i") }).then(
+      (results) => {
+        if (!results || results.length === 0)
+          res
+            .status(404)
+            .json({ message: "No retailers match that search term." });
+        res.json(results);
+      }
+    );
+  },
```

Comment:
if this isn't being used by a route yet, can we comment it out?  There's no way to know if it works correctly without invoking it in a route.